buildpipeline/dotnet-standard-win.json

```diff
@@ -4,75 +4,70 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Install Signing Plugin",
+      "displayName": "git clone",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "30666190-6959-11e5-9f96-f56098202fef",
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "signType": "$(SignType)",
-        "zipSources": "false",
-        "version": "1.0.268",
-        "feedSource": "https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json"
+        "filename": "$(PB_Git)",
+        "arguments": "clone $(PB_VsoCorefxGitUrl) standard",
+        "workingFolder": "",
+        "failOnStandardError": "false"
       }
     },
     {
       "enabled": true,
-      "continueOnError": true,
+      "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Run script build.cmd",
+      "displayName": "git checkout",
       "timeoutInMinutes": 0,
       "task": {
-        "id": "bfc8bf76-e7ac-4a8c-9a55-a944a9f632fd",
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
         "versionSpec": "1.*",
         "definitionType": "task"
       },
       "inputs": {
-        "filename": "build.cmd",
-        "arguments": "-OfficialBuildId=$(OfficialBuildId) -- /p:SignType=real /p:MicroBuild_SigningEnabled=true",
-        "modifyEnvironment": "false",
-        "workingFolder": "",
+        "filename": "$(PB_Git)",
+        "arguments": "checkout $(SourceVersion)",
+        "workingFolder": "standard",
         "failOnStandardError": "false"
       }
     },
     {
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "NuGet Publisher ",
```

Comment:
You're quick!  Please ignore the commit. I'm trying to test the changes.  I'll let you know when ready for review.